agent/src/main/java/com/c8y/ms/templates/agent/App.java

```diff
@@ -0,0 +1,23 @@
+package com.c8y.ms.templates.agent;
+
+import org.springframework.boot.SpringApplication;
+
+import com.cumulocity.microservice.autoconfigure.MicroserviceApplication;
+import com.cumulocity.microservice.context.annotation.EnableContextSupport;
+
+/**
+ * Starter class of template microservice. For starting localy please set environment variable C8Y_MICROSERVICE_ISOLATION=MULTI_TENANT
```

Comment:
Remove this statement: For starting localy please set environment variable C8Y_MICROSERVICE_ISOLATION=MULTI_TENANT. 

This is already set in the application.properties